src/audio/openslES/SDL_openslES.c

```diff
@@ -409,29 +410,34 @@ openslES_CreatePCMPlayer(_THIS)
     SLDataFormat_PCM format_pcm;
     SLresult result;
     int i;
+    long sdk_ver = 0;
+    char sdk_ver_str[92] = {'\0'};
+
+    __system_property_get("ro.build.version.sdk", sdk_ver_str);
+    sdk_ver = SDL_atoi(sdk_ver_str);
```

Comment:
You can use SDL_GetAndroidSDKVersion() here

Reply:
Didn't knew about that call... Thanks